javaagent-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/InClassLoaderMatcher.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.api;
```

Comment:
Consider moving it into `internal` package: same as where `BootstrapPackagePrefixesHolder` (which is also tightly coupled with CL instrumentation) resides